.github/config/dev_changelog_configuration copy.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
@hsein-bitar do we want to keep the copy for reference? or it could be deleted now?

Reply:
No I just renamed it to not include 'copy' thanks for spotting that I totally missed it